client/src/components/profile/Profile.css

```diff
@@ -140,4 +151,47 @@
     float: right;
     color: #333333;
     margin: 20px 0;
-}
\ No newline at end of file
+}
+
+/* comment */
+
+.comment-box {
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.1);
+    /* max-width: 400px; */
```

Comment:
can you add here this => `word-wrap:break-word;`, if user type letter without space it will overflow its parent. If you can find another solution also good.

Reply:
:heavy_check_mark:  done